cheroot/_compat.py

```diff
@@ -15,6 +15,21 @@
 except ImportError:
     IS_ABOVE_OPENSSL10 = None
 
+# contextlib.suppress was added in Python 3.4
+try:
+    from contextlib import suppress
+except ImportError:
+    from contextlib import contextmanager
+
+    @contextmanager
+    def suppress(*exceptions):
+        """Return a context manager that suppresses the `exceptions`."""
+        try:
+            yield
+        except Exception as e:
```

Comment:
I'm pretty sure that you can do
```suggestion
        except exceptions as e:
```
and drop the rest.